File: Iniciante/1044 - Múltiplos.py
"""
Autor: Pierre Vieira
Data da submissão: 25/01/2019 01:31:06
"""
linha = input().split(" ")
A, B = linha
a = int(A)
b = int(B)
if a%b == 0 and a > b or b%a == 0 and b > a:
    print("Sao Multiplos")
else:
    print("Nao sao Multiplos")

File: C#/Iniciante/1044 - Múltiplos.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 14/05/2020 09:16:54
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            string[] linha = Console.ReadLine().Split(' ');
            int n1, n2, aux;
            n1 = int.Parse(linha[0]);
            n2 = int.Parse(linha[1]);
            if (n2 < n1) {
                aux = n2;
                n2 = n1;
                n1 = aux;
            }

            Console.WriteLine(n2 % n1 == 0 ? "Sao Multiplos" : "Nao sao Multiplos");
        }
    }
}
